packages/eslint-plugin-office-addins/jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json'],
+};
```

Comment:
Did you check the other attributes we had on the previous code?
[Plugin Custom Functions](https://github.com/OfficeDev/Office-Addin-Scripts/blob/master/packages/eslint-plugin-excel-custom-functions/jest.config.js)

Just because it had some extra configs that could be useful

Reply:
I looked at those and decided to go with the simpler basic config here (notice the other one doesn't us the preset) and add the others as we discover a need for them.